baseframe/templates/baseframe/mui/forms.html.jinja2

```diff
@@ -287,6 +299,10 @@
   <script type="text/javascript">
     $(function() {
   {%- endif %}
+  // Activate select2.js for non-mobile browsers
+  if (!navigator.userAgent.match(/(iPod|iPad|iPhone|Android)/)) {
```

Comment:
We need a better test than this for mobile. Or at least for when select2 is activated.

Reply:
This is an option - https://github.com/hgoebl/mobile-detect.js/

Reply:
Would detecting if the device supports touch events with `Modernizr.touch` suffice?